torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -259,6 +259,18 @@ static PyObject* get_type_of_overloaded_arg(PyObject* obj_or_type) {
   return (PyObject*)Py_TYPE(obj_or_type);
 }
 
+static py::object maybe_get_registered_torch_dispatch_rule(
+    PyObject* torch_api_function,
+    const py::object& torch_dispatch_object) {
+  static auto find_torch_dispatch_rule =
+      py::module_::import("torch._library.simple_registry")
+          .attr("find_torch_dispatch_rule");
```

Comment:
```suggestion
  static const py::handle find_torch_dispatch_rule =
      py::module_::import("torch._library.simple_registry")
          .attr("find_torch_dispatch_rule").release();
```

Reply:
What's the difference?
Also I can fix this up (go back to the original approach you suggested) when we update pybind11 internally (which will hopefully be soon); I want to merge this PR sooner than later because it may have performance implications.

Reply:
`py::handle` is equivalent to `PyObject *`. It will not do `Py_DECREF(obj.m_ptr)` on C++ instance destruction where `py::object` will. `handle = object.release()` will leak the instance to keep it alive on Python side.